com.unity.render-pipelines.high-definition/Editor/Material/Decal/DecalProjectorEditor.cs

```diff
@@ -568,6 +616,8 @@ public override void OnInspectorGUI()
                     ReinitSavedRatioSizePivotPosition();
                 EditorGUI.EndProperty();
 
+                EditorGUILayout.PropertyField(m_ApplyScale, k_ApplyScale);
```

Comment:
As it affect the size of decal projector, should it be first field? Size line is not directly affected but highly depend on this to know if you need to take into account the hierarchy or not.